site/docs/plugins/conversations.md

```diff
@@ -220,6 +221,111 @@ bot.command("start", (ctx) => ctx.conversation.enter("new-name"));
 
 :::
 
+In total, your code should now roughly look like this.
+
+<CodeGroup>
+  <CodeGroupItem title="TypeScript" active>
+
+```ts
+import { Bot, Context, session } from "grammy";
+import {
+  type Conversation,
+  type ConversationFlavor,
+  conversations,
+  createConversation,
+} from "@grammyjs/conversations";
+
+type MyContext = Context & ConversationFlavor;
+type MyConversation = Conversation<MyContext>;
+
+const bot = new Bot<MyContext>("");
+
+bot.use(session({ initial: () => ({}) }));
+bot.use(conversations());
+
+/** Defines the conversation */
+async function greeting(conversation: MyConversation, ctx: MyContext) {
+  // TODO: code the conversation
```

Comment:
Is this intentionally left blank for users to fill in themselves or we just forgot to put the code in there?

Reply:
This is intentionally left blank because this section explains how to import, install, and enter a conversation. Putting any logic into the function would be missing the point, and distract from what you're trying to say.

Examples for conversation builder functions can be found in other places.